hrjobcontract/CRM/Hrjobcontract/Export/Export.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+class CRM_Hrjobcontract_Export_Export {
```

Comment:
yeah probably you could change it to CRM_Hrjobcontract_Export_Converter or maybe CRM_Hrjobcontract_Export_Main